product_docs/docs/pgd/5/routing/proxy.mdx

```diff
@@ -53,7 +53,7 @@ node needs to miss the reply to 3 heartbeats before it's considered disconnected
 The PGD cluster always has at least one top-level group and one data group. PGD elects the write leader for each data group that has the `enable_proxy_routing` and `enable_raft` options set to true. 
 
 The cluster also maintains Proxy configurations for each group. Each configuration has a name and is associated with a group. You can attach Proxy to a top-level group or data group. You can attach multiple proxies to each group.
```

Comment:
In this paragraph, is proxy meant to be capitalized? If the product, it should be written out as PGD Proxy. Otherwise, lower case. Please advise.

Reply:
```suggestion
The cluster also maintains proxy configurations for each group. Each configuration has a name and is associated with a group. You can attach a proxy to a top-level group or data group. You can attach multiple proxies to each group.
```